libbuteosyncfw/common/NetworkManager.cpp

```diff
@@ -110,12 +142,54 @@ void NetworkManager::sessionConnectionTimeout()
 {
     if (!m_errorEmitted && m_networkSession) {
         if (!m_networkSession->isOpen()) {
-            qWarning() << "No network reply received after 10 seconds, emitting session error.";
+            LOG_WARNING("No network reply received after 10 seconds, emitting session error.");
             slotSessionError(m_networkSession->error());
         }
     }
 }
 
+void NetworkManager::slotConfigurationChanged()
+{
+    // wait for 3 secs before update connection status
+    // this avoid problems with connections that take a while to be stabilished
+    m_idleRefreshTimer.start(3000);
+}
+
+void NetworkManager::idleRefresh()
+{
+    FUNCTION_CALL_TRACE;
+    QList<QNetworkConfiguration> activeConfigs = m_networkConfigManager->allConfigurations(QNetworkConfiguration::Active);
+    QNetworkConfiguration::BearerType connectionType = QNetworkConfiguration::BearerUnknown;
+    QString bearerTypeName;
+
+    bool isOnline = activeConfigs.size() > 0;
+    if (isOnline)
+    {
+        // FIXME: due this bug #1444162 on nm the QNetworkConfigurationManager
```

Comment:
Is fixed on wily network-manager (I can remove this workaround already). Did you guys verify if you have the same problem?